network-api/networkapi/wagtailpages/pagemodels/buyersguide/homepage.py

```diff
@@ -314,9 +314,6 @@ def categories_page(self, request, slug):
                                f' | Mozilla Foundation'
         context['template_cache_key_fragment'] = f'{category.slug}_{request.LANGUAGE_CODE}'
 
-        if not category.show_cta:
```

Comment:
As mentioned in my previous comment, we were originally updating the context, however, I think using show_cta  in the template is more readable.

Reply:
This change is necessary anyhow. When the page loads, it already needs to contain the information for all the categories. JS is then used to switch between which category is shown. That means we can not control if the CTA should be shown based on the first category that the template is loaded with. 

(I am not really a fan of how this works and I think this is also a big contributor to how slow the page is).